### PR TITLE
[Site] Margins and paddings

### PR DESCRIPTION
**Pull request by orta**
Tweaks margins and paddings here and there

### CODE REVIEW

File: website/pages/en/index.js [hunk 1]
```diff
@@ -99,9 +98,6 @@ const TldrSection = ({ language }) => (
             <li>No need to discuss style in code review</li>
             <li>Saves you time and energy</li>
           </ul>
-          <Button href={"/docs/" + language + "/why-prettier.html"}>
-            Why Else?
-          </Button>
```
Comment (orta) [inline website/pages/en/index.js]: I take this one back, I want this link in here.

File: website/data/languages.yml [hunk 2]
```diff
@@ -37,3 +37,4 @@
   - "[PHP](https://github.com/prettier/prettier-php)"
   - "[Swift](https://github.com/prettier/prettier-swift)"
   - "[Java](https://github.com/thorbenvh8/prettier-java)"
+  - "[PostgreSQL](https://github.com/benjie/prettier-plugin-pg-sql/)"
```
Comment (ikatyang) [inline website/data/languages.yml:40]: There is also a Ruby one: https://github.com/iamsolankiamit/prettier-ruby, but I'm not sure if we should we put unofficial plugins here.

**Comment by ikatyang** [comment]
Maybe we should truncate the id?

image

File: website/data/languages.yml (hunk 2 above)
Comment (orta) [reply website/data/languages.yml:40]: Personally, I'm in favor of it - mainly from the perspective of highlighting that there's interesting things happening and to get people to work together. I do see the point of blessing things too early though.

**Comment by orta** [comment]
I wonder about using splitting on character for those cases, but that feels a bit meh. It is better than a broken layout though.

**Comment by sompylasar** [comment]
@orta Change to smaller icons (and font size?), like in the language support section.

**Comment by azz** [comment]
We already do, just might need to decrease the media breakpoint

**Pull request merged**